data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1226 @@
+		never
+	on enter "Vorsuke"
+		fail
+
+	npc
+		to spawn
+			has "provoked military"
+		on kill
+			set "yvx killed"
+		government "Gegno Vi (Combative)"
+		personality heroic vindictive ramming daring staying launching
+		system "Vorsuke"
+		ship "Rhyolite" "Imtydei"
+	npc
+		to spawn
+			has "provoked military"
+		government "Gegno Vi (Combative)"
+		personality heroic vindictive staying launching
+		system "Vorsuke"
+		ship "Schist" "Ess Ouec"
+	npc
+		to spawn
+			has "provoked military"
+		government "Gegno Vi (Combative)"
+		personality heroic vindictive staying launching
+		system "Vorsuke"
+		ship "Schist" "Ess Utchec"
+	npc
+		to spawn
+			has "provoked military"
+		government "Gegno Vi (Combative)"
+		personality heroic vindictive staying entering
+		system "Vorsuke"
+		fleet
+			names "gegno vi"
+			variant
+				"Hornfel (Heavy Weapons)" 1
+				"Eclogite" 2
+
+
+mission "Foreign Spectator I"
+	invisible
+	landing
+	to offer
+		has "First Contact: Gegno Scin: offered"
+		not "Gegno Genocide Defense: offered"
+	to complete
+		never
+	on enter "Eogho"
+		fail
+		"gegno: encounters" ++
+		conversation
+			`As you enter the system, you notice that there is both a Vi and a Scin fleet at the center, although neither of them is engaging the other. Instead, they are observing a pair of ships, one from each side, that appear to be challenging one another. Maybe this is some sort of ceremonial event between the Gegno factions. It's probably best not to interfere.`
+
+	npc
+		on provoke
+			set "interrupted arena"
+		government "Gegno Vi"
+		personality heroic vindictive ramming staying uninterested
+		system "Eogho"
+		ship "Eclogite" "Tschelgi Dueyu"
+	npc
+		on provoke
+			set "interrupted arena"
+		government "Gegno Scin"
+		personality heroic vindictive staying uninterested
+		system "Eogho"
+		ship "Felsic" "Tschelgi Dueich"
+	npc
+		on provoke
+			set "interrupted arena"
+		government "Gegno Vi (Neutral)"
+		personality heroic vindictive staying uninterested swarming
+		system "Eogho"
+		fleet
+			names "gegno vi"
+			variant
+				"Hornfel" 2
+				"Eclogite"
+				"Slate"
+	npc
+		on provoke
+			set "interrupted arena"
+		government "Gegno Scin (Neutral)"
+		personality heroic vindictive staying uninterested swarming
+		system "Eogho"
+		fleet
+			names "gegno scin"
+			variant
+				"Coesite" 2
+				"Tridymite"
+				"Mica" 2
+
+
+mission "Brief Bystander"
+	invisible
+	landing
+	destination "Esstch"
+	to offer
+		has "First Contact: Gegno Scin: offered"
+		not "Gegno Genocide Defense: offered"
+	on complete
+		"gegno: encounters" ++
+		conversation
+			`The presence of scattered recording instruments on the surface of this planet contradicts what little interest the Gegno appear to be giving it nowadays, despite its location within the Gegno's home system of Heutesl. Still, since this planet isn't strictly owned by the Gegno, it makes for a convenient place to stop, and they seem to neither notice nor care that you have landed here.`
```

Comment:
The difficulty here is that the player would have seen those scattered recording instruments on previous visits. This sentence went through iteration, as it was initially written to assume that the player had landed here before. Now, it's supposed to be neutral to whether or not the player has already been here.

Reply:
Ah. Maybe something like:
```suggestion
			`The presence of scattered recording instruments on the surface of this planet suggest that the Gegno are keeping an attentive eye on it. However, and despite its location within the Gegno's home system of Heutesl, you find no other sign of any interest on the part of the Gegno. Since they don't strictly own this planet, it makes for a convenient place to stop, and the Gegno seem to neither notice nor care that you have landed here.`
```

Reply:
Hmmmm... What about:

```suggestion
			`The presence of scattered old recording instruments on the surface of this planet seems to contradict the apparent disinterest with which the Gegno appear to be treating it at present; despite its location within the Gegno's home system of Heutesl, the planet is apparently deserted. Still, since this planet isn't strictly controlled by the Gegno, it makes for a convenient place to stop, and they seem to neither notice nor care that you have landed here.`
```

Reply:
Re Moctave's example - the Gegno are certainly not keeping an eye on it.

Rov - breaking it off from a period makes the first half of the paragraph one long run-on sentence.

Reply:
It's not a run-on sentence, because a semi-colon is breaking up the two independent clauses. I chose to use a semi-colon rather than a period because the two thoughts are linked. The first half is telling you that there is something contradictory about the presence of recording instruments and the apparent disinterest. The second half is following up by explaining that even though the planet is in their home system, it's rather deserted. These two statements go together well imo.

If you mean that it just looks way too long visually, then you could swap the semi-colon for a period. But I personally think it reads better with the semi-colon.

Reply:
I still think the second option is a bunch of wordiness that isn't otherwise needed. I'd cut "at present", leave a period after "it", etc..

Reply:
```suggestion
			`The presence of scattered old recording instruments on the surface of this planet seems to contradict the apparent disinterest with which the Gegno appear to be treating it. Despite its location within the Gegno's home system of Heutesl, the planet is apparently deserted. Still, since this planet isn't strictly controlled by the Gegno, it makes for a convenient place to stop, and they seem to neither notice nor care that you have landed here.`
```
Like this?